### PR TITLE
Spawnmanager

### PR DESCRIPTION
**Pull request by Jamelele**
Just a few tweaks to the spawnmanager documentation which I didn't do the first time around

### CODE REVIEW

File: content/docs/resources/spawnmanager/functions/addSpawnPoint.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ local mySpawnPoint = exports.spawnmanager:addSpawnPoint({
     y = 197.7201,
     x = 466.8401,
     heading = 291.71,
-    model = 1631478380,
+    model = GetHashKey('a_m_m_farmer_01'),
```
Comment (Zeemahh) [inline content/docs/resources/spawnmanager/functions/addSpawnPoint.md]:
```suggestion
    model = `a_m_m_farmer_01`,
```

File: content/docs/resources/spawnmanager/functions/removeSpawnPoint.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ local mySpawnPoint = exports.spawnmanager:addSpawnPoint({
     y = 197.7201,
     x = 466.8401,
     heading = 291.71,
-    model = 1631478380
+    model = GetHashKey('a_m_m_farmer_01')
```
Comment (Zeemahh) [inline content/docs/resources/spawnmanager/functions/removeSpawnPoint.md]: Same as above.

File: content/docs/resources/spawnmanager/functions/spawnPlayer.md [hunk 3]
```diff
@@ -38,4 +38,17 @@ exports.spawnmanager:spawnPlayer()
...
+```
+
+```lua
+-- Spawns the player by passing a table as a spawnpoint
+exports.spawnmanager:spawnPlayer({
+    x = 466.8401,
+    y = 197.7201,
+    z = 111.5291,
+    heading = 291.71,
+    model = GetHashKey('a_m_m_farmer_01')
```
Comment (Zeemahh) [inline content/docs/resources/spawnmanager/functions/spawnPlayer.md]:
```suggestion
    model = `a_m_m_farmer_01`
```

File: content/docs/resources/spawnmanager/functions/setAutoSpawnCallback.md [hunk 4]
```diff
@@ -22,7 +22,32 @@ function callback
...
 exports.spawnmanager:setAutoSpawnCallback(function()
-    -- todo fancy spawning stuff
+    if firstSpawn then
+        -- Spawn them at the spawnpoint
+        exports.spawnmanager:spawnPlayer({
+            x = 466.8401,
+            y = 197.7201,
+            z = 111.5291,
+            heading = 291.71,
+            model = GetHashKey('a_m_m_farmer_01'),
```
Comment (Zeemahh) [inline content/docs/resources/spawnmanager/functions/setAutoSpawnCallback.md]:
```suggestion
            model = `a_m_m_farmer_01`,
```

Comment (Jamelele) [reply content/docs/resources/spawnmanager/functions/setAutoSpawnCallback.md]: I later changed this to just a string in 359eeb3, as spawnmanager will calculate the hash anyway.

**Pull request merged**